packages/core/src/lib/toast/toast-container.svelte

```diff
@@ -28,14 +27,13 @@ $: pageIsVisible.set(visibilityState === 'visible');
 <div
   role="status"
   aria-label="Toasts"
-  class="pointer-events-auto fixed bottom-0 left-1/2 top-auto z-max -translate-x-1/2 transform overflow-hidden p-1.5"
+  class="pointer-events-auto fixed bottom-0 left-1/2 top-auto z-max -translate-x-1/2 transform overflow-hidden pb-6"
 >
   <ul class="flex flex-col items-center gap-2">
     {#each $toasts as { id, pause, resume, ...toast } (id)}
       <li
         in:fly={{ y: 256 }}
-        out:fade
-        animate:flip={{ duration: 200 }}
```

Comment:
Removal of `flip` breaks the gradual shift of old toasts as new ones come in; it causes them to shift instantaneously, which is not what we want